src/en/news/blog/2025/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Rook Ceph Installation on Minikube for Mac M1 with Docker Driver
```

Comment:
Are you sure you want to maintain this doc in the Ceph repo? You might consider linking more to Rook docs such as the developer guide and quickstart, and having only the content in this doc that is differentiated from the Rook docs. This way if there are Rook updates in the future, this guide would less likely be broken. 

For example, in the Rook docs we have a developer guide on minikube as seen [here](https://rook.io/docs/rook/latest-release/Contributing/development-environment/). Or

Reply:
@travisn this PR is not part of ceph docs. it is a blog post

do you also want a fix to the rook docs?
is the current suggestion for mac M1, using qemu + extra_disk working?

Reply:
> do you also want a fix to the rook docs?

which fix? 

> is the current suggestion for mac M1, using qemu + extra_disk working?

Yes this is working, I actually use this option myself